analyzer/source.c

```diff
@@ -1310,8 +1310,8 @@ suscan_source_type_config_helper_str_to_format(const char *format)
   if (format != NULL) {
     if (strcasecmp(format, "AUTO") == 0)
       return SUSCAN_SOURCE_FORMAT_AUTO;
-    else if (strcasecmp(format, "RAW") == 0)
-      return SUSCAN_SOURCE_FORMAT_RAW;
+    else if (strcasecmp(format, "RAW_FLOAT32") == 0)
```

Comment:
These strings are used in configuration files to specify different capture file types. If we replace `"RAW"` by `"RAW_FLOAT32"`, existing configuration files will stop working. Although I prefer the new naming and it would not be critical as it would fallback to `SUSCAN_SOURCE_FORMAT_AUTO`, I think we should ensure backward compatibility by accepting `"RAW"` with identical meaning to `"RAW_FLOAT32"`.

Reply:
Makes sense. Added the check for "RAW".

I suppose a side-effect is that if the config file ever gets rewritten (I have no idea how or when that'd happen), it'll "convert" all "RAW"s to "RAW_FLOAT32" making the config file no longer fully compatible with older code.  (But the old code should fall back to AUTO, and everything should still work.)